elyra/api/elyra.yaml

```diff
@@ -362,9 +362,7 @@ paths:
                 schema:
                   type: object
                   description: A dictionary of property dictionaries. Each property dictionary consists of a key for each instance found and its value (if applicable), or, if parsing fails, is empty.
```

Comment:
I think we should shorten this description to something more like `A dictionary of properties.`, then add descriptions on each of the properties in the reference itself since their types now vary (dictionary vs. list).

The current description is obsolete as we raise exceptions on failures and empty sets are only returned when the type of file is not recognized - which I'm not sure needs to mentioned here.

Reply:
Yeah, I just caught that too! Thoughts on the below descriptions? The `env_vars` one is still perhaps a bit too wordy.

```
env_vars:
          type: object
          description: A dictionary with a key for each environment variable name found and a value corresponding to the variable value or null.
          additionalProperties:
            type: string
inputs:
          type: array
          description: A list of input file names.
          items:
            type: string
outputs:
          type: array
          description: A list of output file names.
          items:
            type: string
```

Reply:
That does seem wordy, but it's difficult.  I think what you have is fine.